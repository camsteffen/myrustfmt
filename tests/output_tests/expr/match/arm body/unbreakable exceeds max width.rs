// test-kind: before-after
// max-width: 20

fn test() {
    match x {
        _ => aaaaaaaaaa,
    }
}

// :after:

fn test() {
    match x {
        _ => {
            aaaaaaaaaa
        }
    }
}
