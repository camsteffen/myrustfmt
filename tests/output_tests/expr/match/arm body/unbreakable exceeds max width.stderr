Max width exceeded at line 4
